tests/Executor/ExecutorTest.php

```diff
@@ -284,22 +284,6 @@ public function testProvidesInfoAboutCurrentExecutionState() : void
 
         Executor::execute($schema, $ast, $rootValue, null, ['var' => '123']);
 
-        self::assertEquals(
-            [
-                'fieldName',
-                'fieldNodes',
-                'returnType',
-                'parentType',
-                'path',
-                'schema',
-                'fragments',
-                'rootValue',
-                'operation',
-                'variableValues',
-            ],
-            array_keys((array) $info)
-        );
-
```

Comment:
Why deleting this? Can't you just add `queryPlan` key to this list?

Reply:
`queryPlan` is declared private because it should only be loaded lazy when you call `lookAhead`
When you cast the `resolveInfo` to array now it cpontains some binary string instead of `queryPlan` because it is private.
Furthermore i really saw no value added by this test, because every key mentionied there would be used for own assertions, so the test will fail anyway if you remove one of the public properties.

Reply:
Sorry, missed your reply. Makes sense.